<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="author" content="Peter Ralph   University of Oregon   Institute of Ecology and Evolution">
  <title>What’s the effect of dispersal?  How gene flow depends on ecology.</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/reset.css">
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
  </style>
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/theme/simple.css" id="theme">
  <style type="text/css">

  .reveal { font-size: 30px; }

  .reveal h1 { font-size: 1.5em; } 

  .reveal h2 { font-size: 1.2em; } 

  .reveal .slides { text-align: left; }

  .reveal .slides figure { text-align: center; }

  .centered p { text-align: center; }

  /* apply this to the columns div */
  div.vcentered { display: flex; align-items: center; }

  .caption p { font-size: 75%; }

  .greyed { color: #4444; }

  div.floatright { float: right; text-align: right; }

  .reveal figcaption { display: none; }

  .reveal img { border: none; }

  .reveal section img { border: none; }

  .MathJax_Display { margin: 0; }

  img.finger { vertical-align: text-top; }

  .smaller { font-size: 50%; }
  .smallish { font-size: 80%; }

  .hide { opacity: 0; }

  </style>

  <div style="display: none">
  \[
  \usepackage{amsmath}
  \usepackage{amssymb}
  \usepackage{xcolor}

  \newcommand{\R}{\mathbb{R}}
  \newcommand{\Q}{\mathbb{Q}}
  \newcommand{\Z}{\mathbb{Z}}
  \newcommand{\N}{\mathbb{N}}
  \newcommand{\C}{\mathbb{C}}

  \renewcommand{\P}{\mathbb{P}}
  \newcommand{\E}{\mathbb{E}}

  \newcommand{\var}{\mathop{\mbox{Var}}}
  \newcommand{\cov}{\mathop{\mbox{cov}}}
  \newcommand{\median}{\mathop{\mbox{median}}}
  %\newcommand{\det}{\mathop{\mbox{det}}}
  \newcommand{\supp}{\mathop{\mbox{supp}}}
  \newcommand{\sgn}{\mathop{\mbox{sgn}}}

  \newcommand{\conv}{\mathop{\mbox{conv}}}
  \newcommand{\deq}{\stackrel{\scriptscriptstyle{d}}{=}}
  \newcommand{\dcv}{\stackrel{\scriptscriptstyle{d}}{\longrightarrow}}

  \]
  </div>
</head>
<body>
  <div class="reveal">
    <div class="slides">

<section id="title-slide">
  <h1 class="title">What’s the effect of dispersal?<br/> How gene flow depends on ecology.</h1>
  <p class="author">Peter Ralph <br/> University of Oregon <br/> Institute of Ecology and Evolution</p>
  <p class="date">5th Workshop Probability and Evolution<br/><em>slides:</em> <a href="https://github.com/petrelharp/cirm_2021_talk">github:petrelharp/cirm_2021_talk</a></p>
</section>

<section class="slide level2">

<div class="columns">
<div class="column">
<p><strong>Joint work with:</strong></p>
<p>Alison Etheridge</p>
<p>Tom Kurtz</p>
<p>Ian Letter</p>
<p>Terence Tsui</p>
</div><div class="column" style="width:50%;">
<p><img data-src="figs/coauthors.png" /></p>
</div>
</div>
</section>
<section id="section" class="slide level2">
<h2><!--
------------------

How can genetic data help us predict the *future*
evolution of populations?
--></h2>
<figure>
<img data-src="figs/xkcd_natural_parenting.png" alt="https://xkcd.com/674/" /><figcaption aria-hidden="true">https://xkcd.com/674/</figcaption>
</figure>
<p><strong>title text:</strong> <em>“On one hand, every single one of my ancestors going back billions of years has managed to figure it out. On the other hand, that’s the mother of all sampling biases.”</em></p>
<div class="caption floatright">
<p><em><a href="https://xkcd.com/674/">xkcd:674</a></em></p>
</div>
</section>
<section>
<section id="the-problem-of-geography" class="title-slide slide level1">
<h1>The “problem” of geography</h1>
<figure>
<img data-src="figs/france-geology.png" alt="geological map of france" /><figcaption aria-hidden="true">geological map of france</figcaption>
</figure>
</section>
<section id="some-popgen-in-space" class="slide level2">
<h2>Some PopGen in Space</h2>
<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li><a href="http://digital.library.adelaide.edu.au/coll/special/fisher/152.pdf">Fisher 1937</a> <!-- The wave of advance of advantageous genes --></li>
<li><a href="http://www.genetics.org/cgi/reprint/28/2/114">Wright 1943</a> <!-- Isolation by Distance --></li>
<li><a href="https://wellcomecollection.org/works/msfaxgkw">Malécot 1948</a> <!-- The mathematics of heredity --></li>
<li><a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1469-1809.1957.tb01867.x/abstract">Maruyama 1972</a> <!-- Distribution of gene frequencies in a geographically structured population.  --></li>
<li><a href="http://www.genetics.org/cgi/content/abstract/75/4/733">Slatkin 1973</a> <!-- Gene flow and selection in a cline --></li>
<li><a href="http://www.jstor.org/stable/2459700">Felsenstein 1975</a> <!-- A Pain in the Torus --></li>
<li><a href="http://www.genetics.org/content/80/3/595.abstract">Nagylaki 1975</a> <!-- Conditions for the existence of clines --></li>
<li><a href="https://doi.org/10.2307%2f1426386">Sawyer 1977</a> <!-- Asymptotic properties of the equilibrium probability of identity in a geographically structured population --></li>
<li><a href="http://dx.doi.org/10.1038/hdy.1979.87">Barton 1979</a> <!-- The dynamics of hybrid zones --></li>
<li><a href="http://projecteuclid.org/euclid.kjm/1250522276">Shiga 1980</a> <!-- An interacting system in population genetics --></li>
<li><a href="https://academic.oup.com/genetics/article-abstract/145/4/1219/6018124">Rousset &amp; Leblois 1997</a> <!-- Likelihood-Based Inferences under Isolation by Distance --></li>
<li><a href="http://www.pnas.org/content/108/5/1783.abstract">Hallatschek 2011</a> <!-- The noisy edge of traveling waves --></li>
<li><a href="http://stacks.iop.org/1742-5468/2013/i=01/a=P01002">Barton, Etheridge &amp; Véber 2013</a> <!-- Modelling evolution in a spatial continuum --></li>
</ul>
</div><div class="column" style="width:50%;">
<figure>
<img data-src="figs/popgen-in-space.jpeg" alt="some white dudes in space suits" /><figcaption aria-hidden="true">some white dudes in space suits</figcaption>
</figure>
<div class="floatright caption">
<p><em>image by Graham Coop</em></p>
</div>
</div>
</div>
<!--
## Waldo Tobler's First Law

> Everything is related to everything else, but near things are more related than distant things.

. . .

![spatial pedigree](figs/spatial_pedigree.png)

::: {.floatright .caption}
*[Bradburd & R 2019](https://www.annualreviews.org/doi/abs/10.1146/annurev-ecolsys-110316-022659)*
:::
-->
</section>
<section id="isolation-by-distance" class="slide level2">
<h2>“Isolation by distance”</h2>
<p>… describes the relationship of genetic distance and geographic distance, where <span class="math display">\[\begin{aligned}
    d(i,j)
    &amp;= \text{(genetic distance between individuals $i$ and $j$)} \\
    &amp;= \frac{\text{number of nucleotide differences}}{\text{length of the genome}} .
\end{aligned}\]</span></p>
</section>
<section class="slide level2">

<p>The <em>Wright-Malecót formula</em> says that in 2D: <span class="math display">\[\begin{aligned}
   \mathbb{E}[1 - d(i, j)]
   &amp;\approx
   \frac{1}{\mathcal{N} + \log(\sigma_e / (\kappa \sqrt{2\mu}))}
    K_0\left( \frac{\sqrt{2\mu}|x_i-x_j|}{\sigma_e} \right) .
\end{aligned}\]</span> (see e.g., Barton, Depaulis &amp; Etheridge (2002); Smith &amp; Weissman 2020)</p>
<div class="fragment">
<div class="floatright">
<p><img data-src="figs/finger_right.png" class="finger" style="width:30.0%" /> <strong>But:</strong> what’s <span class="math inline">\(\sigma_e\)</span>?</p>
</div>
</div>
</section>
<section id="section-1" class="slide level2">
<h2></h2>
<figure>
<img data-src="figs/lineages-hitting-time-divergence.png" alt="IBD for tortoises" /><figcaption aria-hidden="true">IBD for tortoises</figcaption>
</figure>
</section>
<section id="tracing-back-lineages" class="slide level2">
<h2>Tracing back lineages</h2>
<p>The <em>lineage</em> of a bit of modern genome is <span class="math display">\[\begin{aligned}
   L_t = \text{(location of the genetic ancestor at time $t$ ago)} .
\end{aligned}\]</span></p>
<div class="fragment">
<p>Statistical properties of <span class="math inline">\(L\)</span> tell us about:</p>
<ol type="1">
<li><p>gene flow</p></li>
<li><p>long-term fitness</p></li>
<li><p><span class="math inline">\(\sigma_e\)</span></p></li>
</ol>
</div>
</section>
<section id="questions" class="slide level2">
<h2>Questions:</h2>
<p>The <em>dispersal distance</em> is <span class="math display">\[\begin{aligned}
\sigma^2 
    &amp;= \text{(mean squared displacement} \\
    &amp;\qquad \text{between parent and offspring)}
\end{aligned}\]</span> and the <em>effective dispersal distance</em> is <span class="math display">\[\sigma^2_e = \lim_{t \to \infty} \frac{1}{t} \mathbb{E}[L_t^2] .\]</span></p>
<div class="fragment">
<ol type="1">
<li>What determines the sign of <span class="math display">\[\sigma_e^2 - \sigma^2 ? \]</span></li>
</ol>
</div>
<div class="fragment">
<ol start="2" type="1">
<li>How <em>does</em> <span class="math inline">\(L_t\)</span> move, anyhow?</li>
</ol>
</div>
</section></section>
<section>
<section id="a-discrete-model" class="title-slide slide level1">
<h1>A discrete model</h1>
<div class="columns vcentered">
<div class="column" style="width:50%;">
<ul>
<li><p>discrete space and time,</p></li>
<li><p>varying population sizes,</p></li>
<li><p>nonoverlapping generations, with movement only at <em>reproduction</em>.</p></li>
</ul>
</div><div class="column" style="width:50%;">
<figure>
<img data-src="figs/flower.png" alt="Drummond’s anemone" /><figcaption aria-hidden="true">Drummond’s anemone</figcaption>
</figure>
</div>
</div>
</section>
<section id="section-2" class="slide level2">
<h2></h2>
<p>Suppose: <span class="math display">\[\begin{aligned}
    N(x,t)
        &amp;= \text{(number of individuals $t$ generations ago at $x$)} \\
    m(x, y)
        &amp;= \text{(proportion of seeds from $x$ that go to $y$)}
\end{aligned}\]</span></p>
<p>… and surviving individuals are uniform choices out of available seeds.</p>
<div class="fragment">
<p>and so <span class="math display">\[\begin{aligned}
    \mathbb{P}\{ L_t = y \;|\; L_{t-1} = x \}
        &amp;= \frac{ N(y,t) m(y, x) } { \sum_z N(z,t) m(z, x) } .
\end{aligned}\]</span></p>
</div>
<div class="fragment">
<p><em>Assume</em> <span class="math inline">\(L_t\)</span> is Markov given <span class="math inline">\(N\)</span> (generally requires <span class="math inline">\(N\)</span> to be large).</p>
</div>
</section></section>
<section>
<section id="patchy-habiat-consider-a-weed" class="title-slide slide level1">
<h1>Patchy habiat: consider a weed</h1>

</section>
<section class="slide level2">

<p>Assume a 1D grid of populations, and</p>
<ol type="1">
<li>two habitat types: <span class="math inline">\(N(x,t) \in \{G, B\}\)</span>, with <span class="math inline">\(G \gg B\)</span>,</li>
</ol>
<div class="fragment">
<ol start="2" type="1">
<li>For each <span class="math inline">\(t\)</span>, the environment <span class="math inline">\((\ldots, N(x-1, t), N(x, t), N(x+1, t), \ldots)\)</span> is a stationary Markov chain with transition matrix</li>
</ol>
<p><span class="math display">\[\begin{aligned}
    P = \begin{bmatrix}
      1-p &amp; p \\
      q &amp; 1-q \\
     \end{bmatrix} .
\end{aligned}\]</span></p>
</div>
<div class="fragment">
<ol start="3" type="1">
<li>The environment at each time <span class="math inline">\(t\)</span> is <em>independent</em> of everything else.</li>
</ol>
</div>
</section>
<section class="slide level2">

<p>And,</p>
<ol start="4" type="1">
<li><span class="math inline">\(m(x,y) = 1/3\)</span> if <span class="math inline">\(|x-y| \le 1\)</span>.</li>
</ol>
<div class="fragment">
<p><span class="math display">\[\begin{aligned}
    \text{Note: forwards dispersal has } \sigma^2 = \frac{2}{3} .
\end{aligned}\]</span></p>
</div>
<div class="fragment">
<p>Then <span class="math inline">\(L_{t+1} \; | L_t = x\)</span>:</p>
<ul>
<li>chooses a “good” patch uniformly from <span class="math inline">\(\{x-1, x, x+1\}\)</span>, if any;</li>
<li>if not, chooses a neighboring patch uniformly.</li>
</ul>
</div>
</section>
<section id="what-is-sigma2_e" class="slide level2">
<h2>What is <span class="math inline">\(\sigma^2_e\)</span>?</h2>
<table>
<thead>
<tr class="header">
<th>habitat</th>
<th>probability</th>
<th><span class="math inline">\(\mathbb{E}[(L_1 - L_0)^2]\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>GGG</td>
<td><span class="math inline">\(\frac{p}{p+q} (1-q)^2\)</span></td>
<td>2/3</td>
</tr>
<tr class="even">
<td>GGB/BGG</td>
<td><span class="math inline">\(\frac{p}{p+q} (1-q)q\)</span></td>
<td>1/2</td>
</tr>
<tr class="odd">
<td>GBB/BBG</td>
<td><span class="math inline">\(\frac{p}{p+q} q(1-p)\)</span></td>
<td>1</td>
</tr>
<tr class="even">
<td>GBG</td>
<td><span class="math inline">\(\frac{p}{p+q} qp\)</span></td>
<td>1</td>
</tr>
<tr class="odd">
<td>BGB</td>
<td><span class="math inline">\(\frac{q}{p+q} pq\)</span></td>
<td>0</td>
</tr>
<tr class="even">
<td>BBB</td>
<td><span class="math inline">\(\frac{q}{p+q} (1-p)^2\)</span></td>
<td>2/3</td>
</tr>
</tbody>
</table>
<div class="fragment">
<p><span class="math display">\[\begin{aligned}
    \mathbb{E}[(L_1 - L_0)^2]
    =
    \frac{2}{3} + \frac{ pq } {3 (p + q) } \left(1 - (p + q)\right) .
\end{aligned}\]</span></p>
</div>
</section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:60%;">
<p><span class="math display">\[\begin{aligned}
    &amp;\sigma^2_e  - \sigma^2  \\
    &amp;\qquad {} = \frac{ pq } {3 (p + q) } \left(1 - (p + q)\right) .
\end{aligned}\]</span> and so <span class="math display">\[\begin{aligned}
    \sigma_e &gt; \sigma \qquad \text{iff} \quad p + q &lt; 1,
\end{aligned}\]</span></p>
<!-- 
i.e., iff

- $P$ has only positive eigenvalues.
- the environment does not oscillate.
-->
</div><div class="column" style="width:40%;">
<p><img data-src="figs/bigsmall_dvar.png" /></p>
</div>
</div>
</section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:60%;">
<p><span class="math display">\[\begin{aligned}
    &amp;\sigma^2_e  - \sigma^2  \\
    &amp;\qquad {} = \frac{ pq } {3 (p + q) } \left(1 - (p + q)\right) .
\end{aligned}\]</span> and so <span class="math display">\[\begin{aligned}
    \sigma_e &gt; \sigma \qquad \text{iff} \quad p + q &lt; 1,
\end{aligned}\]</span></p>
<p><strong>So:</strong> the random environment can <em>either</em> slow down <em>or</em> speed up a lineage</p>
<p>… even with <em>no</em> temporal correlations!</p>
</div><div class="column" style="width:40%;">
<p><img data-src="figs/bigsmall_dvar.png" /></p>
</div>
</div>
</section>
<section id="what-about-temporal-correlations" class="slide level2">
<h2>What about temporal correlations?</h2>
<div class="columns">
<div class="column">
<p>Now suppose</p>
<ol start="2" type="1">
<li><p>The environment is a stationary, reversible Markov process on <span class="math inline">\(\{G, B\}^\mathbb{Z}\)</span>.</p></li>
<li><p>For each <span class="math inline">\(t\)</span>, the environment <span class="math inline">\(N(t, \cdot)\)</span> is a stationary Markov chain with transition matrix as before.</p></li>
</ol>
</div><div class="column" style="width:50%;">
<figure>
<img data-src="figs/run_BURNIN_1000_NUMGENS_15000_W_300_GAMMA_B_0.2_GAMMA_M_0.1_seed_1302.lineages.png" alt="a simulation" /><figcaption aria-hidden="true">a simulation</figcaption>
</figure>
</div>
</div>
</section>
<section id="what-about-temporal-correlations-1" class="slide level2">
<h2>What about temporal correlations?</h2>
<div class="columns">
<div class="column">
<p>Now suppose</p>
<ol start="2" type="1">
<li><p>The environment is a stationary, reversible Markov process on <span class="math inline">\(\{G, B\}^\mathbb{Z}\)</span>.</p></li>
<li><p>For each <span class="math inline">\(t\)</span>, the environment <span class="math inline">\(N(t, \cdot)\)</span> is a stationary Markov chain with transition matrix as before.</p></li>
</ol>
<p>Numerically: changing temporal correlations can also increase <em>or</em> decrease speed.</p>
</div><div class="column" style="width:50%;">
<p><img data-src="figs/speeds_vs_gamma_m.png" /></p>
</div>
</div>
</section></section>
<section>
<section id="how-do-lineages-move-anyways" class="title-slide slide level1">
<h1>How <em>do</em> lineages move, anyways?</h1>
<p>Next: continuous space (and, time).</p>
</section>
<section id="the-model" class="slide level2">
<h2>The model:</h2>
<ul>
<li><span class="math inline">\(N\)</span>: scaling factor for density</li>
<li><span class="math inline">\(\eta_t\)</span>: point measure with mass <span class="math inline">\(1/N\)</span> for each individual on <span class="math inline">\(\mathbb{R}^d\)</span></li>
<li><span class="math inline">\(\gamma(x, \eta_t)\)</span>: per capita birth rate at <span class="math inline">\(x\)</span></li>
<li><span class="math inline">\(q(x, dy)\)</span>: probability a juvenile disperses to <span class="math inline">\(y\)</span></li>
<li><span class="math inline">\(r(y, \eta_t)\)</span>: juvenile establishment probability at <span class="math inline">\(y\)</span></li>
<li><span class="math inline">\(\mu(x, \eta_t)\)</span>: death rate at <span class="math inline">\(x\)</span></li>
</ul>
<div class="fragment">
<p>The (forwards) <em>dispersal distance</em> is: <span class="math display">\[ \sigma^2 = \int |y-x|^2 q(x, dy) .\]</span></p>
</div>
</section>
<section class="slide level2">

<p>Birth, establishment, and establishment rates depend on local population densities (like Bolker-Pacala):</p>
<ul>
<li><span class="math inline">\(p_\epsilon\)</span>: the heat kernel at time <span class="math inline">\(\epsilon\)</span></li>
<li><span class="math inline">\(p_\epsilon * \eta_t(x)\)</span>: “local” population density at <span class="math inline">\(x\)</span></li>
<li><span class="math inline">\(\sqrt{\epsilon}\)</span>: interaction distance</li>
</ul>
<div class="fragment">
<p>Vital rates at <span class="math inline">\(x\)</span> will depend on <span class="math inline">\(\eta\)</span> through <span class="math inline">\(p_\epsilon * \eta(x)\)</span>.</p>
</div>
</section>
<section id="for-instance" class="slide level2">
<h2>For instance</h2>
<p><em>Mortality increases with crowding:</em> <span class="math inline">\(\gamma\)</span> and <span class="math inline">\(r\)</span> are constant, while <span class="math display">\[\begin{aligned}
    \mu(x, \eta)
    &amp;= \mu(x) \left( 1 - \frac{1}{1 + \exp(p_\epsilon * \eta(x))} \right) .
\end{aligned}\]</span></p>
</section>
<section id="or-for-instance" class="slide level2">
<h2>Or, for instance</h2>
<p><em>Fecundity decreasing with crowding:</em> <span class="math inline">\(\mu\)</span> and <span class="math inline">\(r\)</span> are constant, while <span class="math display">\[\begin{aligned}
    \gamma(x, \eta)
    &amp;= \gamma(x) \left( \frac{1}{1 + \exp(p_\epsilon * \eta(x))} \right) .
\end{aligned}\]</span></p>
</section>
<section id="mean-change-in-eta" class="slide level2">
<h2>Mean change in <span class="math inline">\(\eta\)</span>:</h2>
<p><span class="math display">\[\begin{aligned}
    &amp; \frac{1}{N} \times N \eta(y) \gamma(y, \eta) \; \hphantom{q(y, dx)} &amp;\qquad &amp;\text{(birth at $y$)} \\
    &amp; \hphantom{\frac{1}{N}} \int \; \hphantom{N \eta(y) \gamma(y, \eta)} \; q(y, dx) r(x, \eta) &amp;\qquad &amp;\text{(dispersal to $x$)}
\end{aligned}\]</span></p>
<p><strong>and</strong></p>
<p><span class="math display">\[\begin{aligned}
    &amp; {} - \frac{1}{N} \times N \eta (x) \mu(x, \eta)  &amp; \qquad &amp;\text{(death)}
\end{aligned}\]</span></p>
</section>
<section id="the-mean-measure" class="slide level2">
<h2>The mean measure</h2>
<p>So: for a test function <span class="math inline">\(f\)</span>, <span class="math display">\[\begin{aligned}
    &amp;
    \lim_{t \searrow 0} \frac{1}{t}
        \left. \mathbb{E} \left[
            \int f(x) \eta_{t}(dx) - \int f(x) \eta_0(dx)
        \;|\; \eta_0 = \eta \right] \right\vert_{t=0} \\
    &amp;\qquad
    = \int \int f(x) r(x, \eta) q(y, dx) \gamma(y, \eta) \eta(dy) \\
    &amp;\qquad \qquad {}
    - \int f(x) \mu(x, \eta) \eta(dx) .
\end{aligned}\]</span></p>
<div class="fragment">
<p><span class="math display">\[\begin{aligned}
    &amp;{}
    = \int \left\{ \int \left( f(x) r(x, \eta) 
                              - f(y) r(y, \eta) \right) 
                         q(y, dx) \right\}
             \gamma(y, \eta) \eta(dy) \\
    &amp;\qquad \qquad {}
    + \int f(x)
        \left\{
            r(x, \eta) \gamma(x, \eta)
            - \mu(x, \eta)
        \right\}
      \eta(dx) .
\end{aligned}\]</span></p>
</div>
</section>
<section id="what-about-noise" class="slide level2">
<h2>What about noise?</h2>
<p>Since reproduction produces single offspring,</p>
<p><span class="math display">\[\begin{aligned}
    &amp;
    \lim_{t \searrow 0} \frac{1}{t}
        \left. \mathbb{E} \left[
                \left( \int f(x) \eta_{t}(dx) - \int f(x) \eta_0(dx) \right)^2
            \;|\; \eta_0 = \eta \right] \right\vert_{t=0} \\
    &amp;\quad {}
    = \frac{1}{N}
    \left\{
        \int 
            \int f^2(x) r(x, \eta) q(y, dx)
        \gamma(y, \eta) \eta(dy)
    \right. \\
    &amp;\qquad \qquad \left. {}
    + \int f^2(x) \mu(x, \eta) \eta(dx)
    \right\} 
\end{aligned}\]</span></p>
<div class="fragment">
<p><span class="math display">\[\begin{aligned}
    {}
    \propto \frac{1}{N} .
    \hphantom{
            \left\{ \int \int f^2(x) r(x, \eta) q(y, dx) \gamma(y, \eta) \eta(dy) \right\}
    }
\end{aligned}\]</span></p>
</div>
</section></section>
<section>
<section id="diffusion-limits-heuristics" class="title-slide slide level1">
<h1>Diffusion limits: heuristics</h1>
<p><img data-src="figs/scales_diagram.png" /></p>
</section>
<section class="slide level2">

<p>As <span class="math inline">\(N \to \infty\)</span>, also rescale <em>time</em> by <span class="math inline">\(\theta\)</span>, and let e.g., <span class="math display">\[  r_\theta(x, \eta) \to r(x, \eta) \qquad \text{as } \theta \to \infty . \]</span></p>
<p>As <span class="math inline">\(\theta \to \infty\)</span>, to see <em>lineages</em> moving, we take <span class="math inline">\(\sigma = 1/\sqrt{\theta}\)</span>, so that <span class="math display">\[
    \theta \int (g(y) - g(x)) q_\theta(x, dy) \to \Delta g(x) ,
\]</span></p>
<div class="fragment">
<p>and population density changes on a time scale of <span class="math inline">\(\theta\)</span> generations: <span class="math display">\[
    \theta\left(
        r_\theta(x, \eta) \gamma_\theta(x, \eta)
        - \mu_\theta(x, \eta)
    \right)
        \to F(x, \eta) .
\]</span></p>
</div>
</section>
<section class="slide level2">

<p>Suppose also the <em>density measure</em> converges:</p>
<p><span class="math display">\[
    \eta \to \Xi \qquad \text{as} \qquad \theta, N \to \infty
\]</span></p>
<p>and so <span class="math display">\[\begin{aligned}
    &amp;
    \lim_{t \searrow 0} \frac{1}{t}
        \left. \mathbb{E} \left[ \int f(x) \eta_{t}(dx) - \int f(x) \eta_0(dx) \;|\; \eta_0 = \eta \right] \right\vert_{t=0} \\
    &amp;\qquad
    \to
        \int \left\{\vphantom{\int}
            \gamma(x, \Xi) \Delta\left(
                    f(\cdot) r(\cdot, \Xi)
                \right)\!(x)
    \right. \\ &amp;\qquad \qquad \qquad \left. \vphantom{\int}
        + f(x) F(x, \Xi) 
    \right\} \Xi(dx) .
\end{aligned}\]</span></p>
</section>
<section id="three-types-of-limits" class="slide level2">
<h2>Three types of limits</h2>
<p>The limit acts like <span class="math display">\[
    ``\; \dot \Xi = r \Delta(\gamma \Xi) + F \Xi . \text{&#39;&#39;}
\]</span> … but recall that the coefficients are “nonlocal”:<br />
<span class="math inline">\(r\)</span> may be a function of <span class="math inline">\(p_\epsilon * \Xi\)</span>.</p>
<div class="fragment">
<p>Some options for <span class="math inline">\(\Xi\)</span>:</p>
<ol type="1">
<li><p>Stochastic, nonlocal coefficients. (superprocess limit)</p></li>
<li><p>Deterministic, nonlocal coefficients.</p></li>
<li><p>Deterministic, local coefficients. (PDE limit)</p></li>
</ol>
</div>
</section>
<section id="superprocess-limit" class="slide level2">
<h2>Superprocess limit:</h2>
<p>Quadratic variation of the limit is nonzero if <span class="math display">\[ \frac{N}{\theta} \to \rho, \]</span> for some <span class="math inline">\(\rho &gt; 0\)</span>.</p>
<div class="fragment">
<p>In other words, since <span class="math inline">\(\sigma = 1/\sqrt{\theta}\)</span>, Wright’s neighborhood size is: <span class="math display">\[\begin{aligned}
    \mathcal{N}
    &amp;:= \text{(mean number of individuals within distance $\sigma$)} \\
    &amp;\propto N \sigma^d 
\end{aligned}\]</span> … which is equal to <span class="math inline">\(\rho\)</span> in <span class="math inline">\(d=2\)</span>.</p>
</div>
</section>
<section id="deterministic-limit-thetan-to-0" class="slide level2">
<h2>Deterministic limit: <span class="math inline">\(\theta/N \to 0\)</span></h2>
<p>If the limiting measure has density <span class="math inline">\(\Xi_t(x) dx\)</span>, then it’s a weak solution to <span class="math display">\[\begin{aligned}
    \frac{d}{dt} \Xi_t(x)
    &amp;=
        r(x, \Xi) \Delta\left(
                \gamma(\cdot, \Xi_t) \Xi_t(\cdot)
            \right)(x)
        + F(x, \Xi_t) \Xi_t(x) .
\end{aligned}\]</span></p>
<div class="fragment">
<p>i.e., <span class="math display">\[\begin{aligned}
    \dot \Xi = r \Delta\left( \gamma \Xi \right) + F \Xi .
\end{aligned}\]</span></p>
</div>
</section>
<section id="pde-limit" class="slide level2">
<h2>PDE limit?</h2>
<p>Recall that e.g., <span class="math display">\[\begin{aligned}
r(x, \Xi)
    = r(p_\epsilon * \Xi(x)) .
\end{aligned}\]</span></p>
<div class="fragment">
<p>… can we also take <span class="math inline">\(\epsilon \to 0\)</span>, getting <span class="math display">\[\begin{aligned}
    \frac{d}{dt} \Xi_t(x)
    &amp;=
        r(\Xi_t(x)) \Delta\left(
                \gamma(\Xi_t(\cdot)) \Xi_t(\cdot)
            \right)\!(x) \\
    &amp;\qquad {}
        + F(\Xi_t(x)) \Xi_t(x) ?
\end{aligned}\]</span></p>
</div>
</section>
<section class="slide level2">

<p>For fixed <span class="math inline">\(\epsilon\)</span>, we have that <span class="math display">\[
    \eta_\epsilon \to \Xi_\epsilon \qquad \text{as } N, \theta \to \infty.
\]</span></p>
<div class="fragment">
<p>We need <span class="math inline">\(p_\epsilon * \eta_\epsilon(x) \to \Xi(x)\)</span>, e.g., <span class="math display">\[\begin{aligned}
    &amp;
    (p_\epsilon * \eta_\epsilon - p_\epsilon * \Xi_\epsilon)
    \hphantom{+ (p_\epsilon * \Xi_\epsilon - p_\epsilon * \Xi)
    + (p_\epsilon * \Xi - \Xi)} \\
    &amp;\hphantom{(p_\epsilon * \eta_\epsilon - p_\epsilon * \Xi_\epsilon)}
    + (p_\epsilon * \Xi_\epsilon - p_\epsilon * \Xi)
    \hphantom{+ (p_\epsilon * \Xi - \Xi) } \\
    &amp;\hphantom{(p_\epsilon * \eta_\epsilon - p_\epsilon * \Xi_\epsilon)
    + (p_\epsilon * \Xi_\epsilon - p_\epsilon * \Xi)}
    + (p_\epsilon * \Xi - \Xi) \\
    &amp;\qquad \to 0 \qquad \text{as} \qquad N, \theta \to \infty, \qquad \epsilon \to 0 .
\end{aligned}\]</span></p>
</div>
</section></section>
<section>
<section id="what-about-the-lineages" class="title-slide slide level1">
<h1>What about the lineages?</h1>

</section>
<section class="slide level2">

<p><strong>Goal:</strong> rescale population densities while <em>retaining the notion of lineages</em>.</p>
<div class="fragment">
<p><strong>Method:</strong> a lookdown construction, following <a href="https://dx.doi.org/10.1214/10-AOP574">Kurtz &amp; Rodrigues 2011</a> and <a href="https://dx.doi.org/10.1214/18-AOP1266">Etheridge &amp; Kurtz 2019</a>.</p>
</div>
<div class="fragment">
<p>… this <em>also</em> gives us tightness for the population processes themselves!</p>
</div>
</section>
<section id="deterministic-nonlocal-limit" class="slide level2">
<h2>Deterministic, nonlocal limit:</h2>
<p><em>Theorem:</em> <!-- Thm 2.9 --> Suppose that for fixed <span class="math inline">\(\epsilon\)</span>, <span class="math display">\[\begin{aligned}
    r(x, \eta) &amp;= r(x), \\
    \gamma_\theta(x, \eta)
        &amp;= \gamma(p_\epsilon * \eta(x))
            + \frac{G(p_\epsilon * \eta(x))}{\theta r(x)} \\
    \mu_\theta(x, \eta)
        &amp;= \gamma(p_\epsilon * \eta(x))
            + \frac{H(p_\epsilon * \eta(x))}{\theta} \\
    q_\theta(x, dy) &amp;= p_{1/\theta}(x, dy) ,
\end{aligned}\]</span> with <span class="math inline">\(\gamma\)</span>, <span class="math inline">\(G\)</span>, <span class="math inline">\(H\)</span>, and <span class="math inline">\(r\)</span> uniformly bounded and Lipschitz continuous, and <span class="math inline">\(0 &lt; r_0 &lt; r(x) \le 1\)</span> twice diff’able. Then a lookdown construction with maximum level <span class="math inline">\(N \to \infty\)</span>, with <span class="math inline">\(\theta \to \infty\)</span> and <span class="math inline">\(\theta / N \to 0\)</span>, converges <!-- in the sense of finite-dimensional distributions --> to a measure-valued process <span class="math inline">\((\eta_t^\infty)_{t \ge 0}\)</span>.</p>
</section>
<section id="theorem-continued" class="slide level2">
<h2>(theorem, continued)</h2>
<p>The limit is a Cox measure with intensity a product of <span class="math inline">\(\Xi_t \times \Lambda\)</span>, and for every continuous, bounded <span class="math inline">\(f : \mathbb{R}^d \to \mathbb{R}_+\)</span>, <span class="math display">\[\begin{aligned}
    &amp; \int f(x) \Xi_t(dx)
      - \int f(x) \Xi_0(dx) \\
    &amp;=
      \int_0^t \int
        \gamma(p_\epsilon * \Xi_s(x))
        \Delta \left( f(\cdot) r(\cdot) \right)(x) \\
    {}&amp;\qquad
        + f(x) \left\{
            G(p_\epsilon * \Xi_s(x))
            - H(p_\epsilon * \Xi_s(x))
        \right\}
      \Xi_s(dx) ds .
\end{aligned}\]</span></p>
</section>
<section id="pde-limit-1" class="slide level2">
<h2>PDE limit</h2>
<p><em>Theorem:</em> <!-- Thm 2.10 --> Suppose that <span class="math inline">\(r(x) = 1\)</span> and <span class="math display">\[\begin{aligned}
    \gamma_\theta(x, \eta)
        &amp;= 1
            + \frac{G(p_\epsilon * \eta(x))}{\theta} \\
    \mu_\theta(x, \eta)
        &amp;= 1
            + \frac{H(p_\epsilon * \eta(x))}{\theta} \\
    q_\theta(x, dy) &amp;= p_{1/\theta}(x, dy) ,
\end{aligned}\]</span> with <span class="math inline">\(G\)</span>, <span class="math inline">\(H\)</span> positive, Lipschitz continuous with <span class="math inline">\(G(1) = H(1)\)</span> and such that <span class="math display">\[
    G(u), H(u)  \le C \left( 1 + u^p \right) .
\]</span></p>
</section>
<section id="theorem-continued-1" class="slide level2">
<h2>(theorem, continued)</h2>
<p>Then a lookdown construction with maximum level <span class="math inline">\(N\)</span>, as <span class="math inline">\(N, \theta \to \infty\)</span>, if <span class="math inline">\(\theta/N \to 0\)</span> and <span class="math display">\[
    \frac{\theta}{N \epsilon^{3dp/2}}
    + \frac{1}{\theta \epsilon^{dp/2}}
    \to 0 ,
\]</span> converges to a measure-valued process <span class="math inline">\((\eta_t^\infty)_{t \ge 0}\)</span>.</p>
<p>The limit is a Cox measure with intensity a product of <span class="math inline">\(\Xi_t \times \Lambda\)</span>, where <span class="math inline">\(\Xi_t\)</span> has a density that is a weak solution to <span class="math display">\[\begin{aligned}
    \frac{d}{dt} \Xi
        &amp;= \Delta \Xi + (G(\Xi) - H(\Xi)) \Xi,
\end{aligned}\]</span> for <span class="math inline">\(x \in \mathbb{R}^d\)</span> and <span class="math inline">\(t &gt; 0\)</span> and appropriate initial conditions.</p>
</section></section>
<section>
<section id="a-note-of-caution" class="title-slide slide level1">
<h1>A note of caution</h1>
<div class="floatright">
<figure>
<img data-src="figs/gilia.jpeg" style="width:40.0%" alt="Gilia Patterson" /><figcaption aria-hidden="true">Gilia Patterson</figcaption>
</figure>
<div class="caption">
<p><em>Gilia Patterson</em></p>
</div>
</div>
</section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:60%;">
<ul>
<li><p>death: <span class="math inline">\(\mu = 0.3\)</span> per generation</p></li>
<li><p>establishment: <span class="math inline">\(r = 0.7\)</span></p></li>
<li><p>dispersal: Gaussian with SD <span class="math inline">\(\sigma\)</span></p></li>
<li><p>local density: in circle of radius <span class="math inline">\(\epsilon\)</span></p></li>
<li><p>reproduction: with <span class="math inline">\(K=2\)</span>, <span class="math inline">\(\lambda=3\)</span>, <span class="math display">\[ \gamma = \frac{\lambda}{1 + \text{(local density)}/K} \]</span></p></li>
<li><p>non-spatial equilibrium density: <span class="math display">\[ K \left( \frac{\lambda}{1 - r} - 1 \right) .\]</span></p></li>
</ul>
</div><div class="column" style="width:40%;">
<figure>
<img data-src="figs/slim_screenshot.png" alt="screenshot of SLiM GUI" /><figcaption aria-hidden="true">screenshot of SLiM GUI</figcaption>
</figure>
<div class="caption">
<p><em>SLiM code at <a href="https://github.com/petrelharp/cirm_2021_talk/gilia_clumping_sim.slim">github.com/petrelharp/cirm_2021_talk</a></em></p>
</div>
</div>
</div>
</section>
<section class="slide level2">

<figure>
<img data-src="figs/gilia_density_results.png" alt="population densities" /><figcaption aria-hidden="true">population densities</figcaption>
</figure>
</section>
<section class="slide level2">

<ul>
<li>dispersal distance <span class="math inline">\(\sigma = 1/\sqrt{\theta} = 3\)</span></li>
<li>interaction distance <span class="math inline">\(\epsilon = 1\)</span></li>
<li>mean # offspring <span class="math inline">\(\gamma \propto (1 + \text{(density)} / K)^{-1}\)</span></li>
</ul>
<div class="fragment">
<figure>
<img data-src="figs/gilia_high_sigma.png" alt="not patchy" /><figcaption aria-hidden="true">not patchy</figcaption>
</figure>
</div>
</section>
<section class="slide level2">

<ul>
<li>dispersal distance <span class="math inline">\(\sigma = 1/\sqrt{\theta} = 0.2\)</span></li>
<li>interaction distance <span class="math inline">\(\epsilon = 1\)</span></li>
<li>mean # offspring <span class="math inline">\(\gamma \propto (1 + \text{(density)} / K)^{-1}\)</span></li>
</ul>
<figure>
<img data-src="figs/gilia_low_sigma.png" alt="not patchy" /><figcaption aria-hidden="true">not patchy</figcaption>
</figure>
</section>
<section class="slide level2">

<div class="centered">
<p><a href="https://doi.org/10.1006%2Fjtbi.1996.0370"><img data-src="figs/sasaki-title.png" alt="Clumped Distribution by Neighbourhood Competition, by A. Sasaki. Journal of Theoretical Biology 186 (4): 415–430 (June 1997)" /></a></p>
<figure>
<img data-src="figs/sasaki-bumps.png" alt="a figure from Sasaki 1997" /><figcaption aria-hidden="true">a figure from Sasaki 1997</figcaption>
</figure>
<div class="floatright caption">
<p><em>Sasaki (1997)</em></p>
</div>
</div>
</section></section>
<section>
<section id="consequences-for-stationary-landscapes" class="title-slide slide level1">
<h1>Consequences for stationary landscapes</h1>

</section>
<section class="slide level2">

<p>If a deterministic, local limit holds, with dispersal <span class="math inline">\(N(m, \sigma^2 I)\)</span>, lineages should move in a stationary population density <span class="math inline">\(n(x) = d\Xi/dx\)</span> as <span class="math display">\[\begin{aligned}
    dL_t = r(L_t) \gamma(L_t) \left\{
            \left(
                2\sigma^2 \nabla \log(n\gamma)(L_t)  - m
            \right) dt
            + \sigma dB_t \right\} .
\end{aligned}\]</span></p>
<div class="fragment">
<p>i.e., as Brownian motion run at speed <span class="math inline">\(\sigma r(y) \gamma(y)\)</span> in the potential <span class="math display">\[\begin{aligned}
    n(y) \gamma(y) e^{-my/(2\sigma^2)} ,
\end{aligned}\]</span> … which has stationary distribution <span class="math display">\[\begin{aligned}
    \frac{n(y)}{r(y)} e^{-my/(2\sigma^2)} .
\end{aligned}\]</span></p>
</div>
</section></section>
<section>
<section id="lineages-in-expanding-populations" class="title-slide slide level1">
<h1>Lineages in expanding populations</h1>

</section>
<section id="travelling-waves" class="slide level2">
<h2>Travelling waves</h2>
<p>Suppose the population density has a traveling wave profile: <span class="math inline">\(\Xi_t(dx) = n(t,x) dx\)</span> with <span class="math display">\[\begin{aligned}
    n(x,t) = w(x - ct),
\end{aligned}\]</span> and a determinstic, local limit holds:</p>
<div class="fragment">
<p>… then <span class="math inline">\(L - ct\)</span> has generator <span class="math display">\[\begin{aligned}
    \phi &amp;\mapsto r(x) \gamma(x) \left\{
        2 \nabla \log(\gamma w)(x) \cdot \nabla \phi(x)
        + \Delta \phi(x)
    \right\} 
        \\ &amp;\qquad {}
    + c \cdot \nabla \phi(x) .
\end{aligned}\]</span></p>
</div>
</section>
<section id="example-pme" class="slide level2">
<h2>Example: PME</h2>
<p>For instance, take the porous medium equation with logistic growth (in 1D): <span class="math display">\[\begin{aligned}
   \partial_t n_t(x) = \partial_x^2 [n_t(x)^2] + n_t(x) (1 - n_t(x)) ,
\end{aligned}\]</span> with stable solution <span class="math display">\[\begin{aligned}
    n_t(x) = \left( 1 - \exp\left( \frac{1}{2} (x - t) \right)\right)_+
\end{aligned}\]</span></p>
<div class="fragment">
<p>To get this, we want <span class="math inline">\(r=1\)</span> and <span class="math display">\[\begin{aligned}
    \gamma(x, n) &amp;= n(x) \\
    \mu(x, n) &amp;= 2 n(x) - 1 .
\end{aligned}\]</span></p>
</div>
</section>
<section class="slide level2">

<p>…so in the stationary frame, the lineage’s generator is <span class="math display">\[\begin{aligned}
    \phi 
    &amp;\mapsto 
    w(x) \left( \phi_{xx} + 4 (\log w)_x \phi_x \right) + \phi_x \\
    &amp;=
    \left(1 - e^{x/2}\right) \phi_{xx}
            + \left(1 - 2 e^{x/2}\right) \phi_x \qquad \text{on } x &lt; 0.
\end{aligned}\]</span></p>
</section>
<section class="slide level2">

<div class="columns">
<div class="column">
<p>The lineage has stationary distribution <span class="math display">\[\begin{aligned}
    \pi(x) \propto e^x \left(1 - e^{x/2}\right)
\end{aligned}\]</span> for <span class="math inline">\(x &lt; 0\)</span>.</p>
<p>… in constrast to the Fisher-KPP.</p>
</div><div class="column" style="width:50%;">
<p><img data-src="figs/pme_dists.png" /></p>
</div>
</div>
</section></section>
<section>
<section id="wrap-up" class="title-slide slide level1">
<h1>Wrap-up</h1>

</section>
<section id="open-questions" class="slide level2">
<h2>Open questions</h2>
<ol type="1">
<li><p>When is <span class="math inline">\(\sigma_e &gt; \sigma\)</span>?</p></li>
<li><p>How badly are lineages non-Markov?</p></li>
<li><p>What can (and can’t) inferences about reverse-time tell us about forwards-time?</p></li>
</ol>
<figure>
<img data-src="figs/locator_spatial_tree_sequence.png" alt="trees in space" /><figcaption aria-hidden="true">trees in space</figcaption>
</figure>
<div class="caption">
<p><em>image by CJ Battey</em></p>
</div>
</section>
<section id="credit" class="slide level2">
<h2>Credit</h2>
<div class="columns">
<div class="column" style="width:40%;">
<ul>
<li>Alison Etheridge</li>
<li>Tom Kurtz</li>
<li>Ian Letter</li>
<li>Terence Tsui</li>
<li>Aaron Smith</li>
<li>Gilia Patterson</li>
</ul>
<p><em>and</em></p>
<ul>
<li>Andy Kern</li>
<li>CJ Battey</li>
<li>Gideon Bradburd</li>
<li>the Kern-Ralph co-lab</li>
</ul>
</div><div class="column" style="width:60%;">
<p><em>Simulation methods:</em></p>
<ul>
<li>Ben Haller (<a href="https://messerlab.org/SLiM">messerlab.org/SLiM</a>)</li>
<li>Jerome Kelleher (<a href="https://tskit.dev/msprime">tskit.dev/msprime</a>)</li>
<li>the tskit consortium (<a href="https://tskit.dev/">tskit.dev</a>)</li>
</ul>
<p><img data-src="figs/slim_logo.png" style="width:50.0%" alt="SLiM logo" /> <img data-src="figs/tskit_logo.png" style="width:45.0%" alt="tskit logo" /></p>
</div>
</div>
</section>
<section id="section-3" class="slide level2" data-background-image="figs/guillemots_thanks.png" data-background-position="center" data-background-size="50%">
<h2 data-background-image="figs/guillemots_thanks.png" data-background-position="center" data-background-size="50%"></h2>
</section></section>
    </div>
  </div>

  <script src="https://unpkg.com/reveal.js@^4//dist/reveal.js"></script>

  // reveal.js plugins
  <script src="https://unpkg.com/reveal.js@^4//plugin/notes/notes.js"></script>
  <script src="https://unpkg.com/reveal.js@^4//plugin/search/search.js"></script>
  <script src="https://unpkg.com/reveal.js@^4//plugin/zoom/zoom.js"></script>
  <script src="https://unpkg.com/reveal.js@^4//plugin/math/math.js"></script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
        // Display the page number of the current slide
        slideNumber: true,
        // Push each slide change to the browser history
        history: true,
        // Transition style
        transition: 'none', // none/fade/slide/convex/concave/zoom
        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [
          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    </body>
</html>
